apps/emqx_plugin_libs/src/emqx_plugin_libs_ssl.erl

```diff
@@ -96,8 +96,10 @@ do_save_file(FilePath, _) when is_list(FilePath) ->
     FilePath;
 do_save_file(_, _) -> "".
 
-do_save_file("", _, _Dir) -> ""; %% ignore
-do_save_file(_, <<>>, _Dir) -> ""; %% ignore
+do_save_file("", _, _Dir) -> ""; %% no filename, ignore
+do_save_file(FileName, <<>>, Dir) ->  %% no content, see if file exists
```

Comment:
Q: which case corresponds to having no content but an existing filepath?  Maybe the user just points to the existing file instead of uploading the cert?